doc/apiv3-documentation.apib

```diff
@@ -2139,6 +2139,84 @@ For more details and all possible responses see the general specification of [Fo
                 "message": "The request body was not empty."
             }
 
+
+# Group Revisions
+
+Revisions are sets of updates to files in the context of repositories linked in OpenProject.
+
+## Linked Properties
+|  Link            | Description                                         | Type          | Constraints | Supported operations |
+|:----------------:| --------------------------------------------------- | ------------- | ----------- | -------------------- |
+| self             | This revision                                       | Revision      | not null    | READ                 |
+| project          | The project to which the revision belongs           | Project       | not null    | READ                 |
+| committer        | The user that committed this revision, if defined   | User          |             | READ                 |
```

Comment:
My first thought for this field name: Why is it not called `author`? Because that is the general name we use in the API for the person that creates something.

My second thought was: Or will it use [git semantics of committer vs author](http://stackoverflow.com/questions/18750808/difference-between-author-and-committer-in-git)? But if that was true more questions would arise, I therefore assume you are talking about the git author here.

**tl; dr:** I think this property should be called `author`